hack/prepare-release-branch.sh

```diff
@@ -0,0 +1,71 @@
+# !/bin/bash
+#
+# Script runs below steps to prepare a new release branch:
+# - Fetch latest main branch from remote origin (it must point to gitbu.com/intel/trusted-certificate-issuer).
+# - Create a new branch with current main HEAD.
+# - Run needed make targets to generate manifests with the new version.
+# - Modify helm chats with new version.
+# - Commit the changes.
+# - And return back to the previous branch.
+#
+set -o pipefail
+set -o errexit
+
+SOURCE=$(dirname "$(readlink -f "$0")")
+REPO_ROOT=$(dirname $SOURCE)
+
+VERSION=
+pwd=$(pwd)
+current_branch=$(git branch --show-current)
+release_branch=
+
+function Usage {
+    echo "Usage:"
+    echo "   $0 --version <semver version>"
+}
+
+for opt in $@
+do
+    case "$opt" in
+    --version)
+        shift ; VERSION=$1 ;;
+    -h | --help)
+        Usage ; exit ;;
+    -*) shift ; echo "Unrecognized option $opt" ;;
+    esac
+done
+
+if [ -z "$VERSION" ]; then
+    echo "ERROR: No release version set."
+    Usage
+    exit
+fi
+
+function Cleanup {
+    git checkout $current_branch
+    if [ ! -z "$release_branch" ] ; then
+        git branch -D "$release_branch"
+    fi
+    cd $pwd
```

Comment:
Use quotes around directory names in case someone has a space in them.

Reply:
Fixed.